content-model/articles.json

```diff
@@ -96,6 +96,32 @@
               }
             }
           },
+          "iframe" : {
+            "type" : "Slice",
+            "fieldset" : "Iframe",
+            "non-repeat" : {
+              "launchButtonText" : {
+                "type" : "Text",
+                "config" : {
+                  "label" : "Launch button text",
+                  "placeholder" : "launch button text"
+                }
+              },
+              "iframeSrc" : {
+                "type" : "Text",
+                "config" : {
+                  "label" : "iframe src",
+                  "placeholder" : "iframe src"
+                }
+              },
+              "previewImage" : {
```

Comment:
Are we definitely going to get these? And if not, what do we show?

Reply:
I guess we're not _definitely_ going to get them, because Prismic. We could either display a black box with a button on top, or maybe have the iframe load straight away if there isn't an image